cmd/miniooni/main.go

```diff
@@ -145,7 +143,15 @@ type logHandler struct {
 }
 
 func (h *logHandler) HandleLog(e *log.Entry) (err error) {
-	s := fmt.Sprintf("<%s> %s", e.Level, e.Message)
+	if e.Level > log.InfoLevel {
+		wrongCounter++
+	}
+	if !globalOptions.verbose && e.Level != log.InfoLevel &&  e.Level != log.FatalLevel {
+		return nil
+	}
```

Comment:
What are you trying to achieve here? You can set the verbosity of `apex/log` directly and we already do that, so I do not fully understand why you are filtering logs by verbosity also in the handler.

Reply:
`apex/log` does this via `level`. We cannot set the level `info` and prevent it from displaying a `warning`.
https://github.com/apex/log/blob/ff0f66940b829dc66c81dad34746d4349b83eb9e/logger.go#L142
Also, if the user uses `-v`, we should not stop displaying the `warning`.

Reply:
I believe there is some misunderstanding in how logging should work?

This is how it should work:

1. no `-v` then you see fatal, error, warning and info logs

2. with `-v` you also see debug logs

Reply:
>with -v you also see debug logs

with `-v` you also see `all` logs

If you set `level` to `warning` then you see `warning`, `error`, and `fatal` logs

Reply:
Yes, that's what I mean with "with `-v` you _also_ see debug logs"

Reply:
But if the above line being without `!globalOptions.verbose && ` and if the user uses `-v`, no `warning` and `error` logs will be displayed for him. Because all `log.Logs` have been redirected to this handler.

Reply:
Because this handle is using `stderr` there is no log redirection. I suggest you try to remove this code, run the program and see whether it's working as intended or not. Maybe I am missing something, but it seems to me these lines are duplicating apex/log functionality and can safely be removed without losing any functionality.